<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>318</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>318</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>318</width>
     <height>512</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>318</width>
     <height>512</height>
    </size>
   </property>
   <widget class="QLabel" name="label_up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>285</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
color:rgb(254, 254, 176);
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_int">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Инт</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_conv">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Конв</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CE">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_H">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Ч</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>М</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_S">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>170</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:rgb(255, 81, 0);
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>С</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_down">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>285</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
color:rgb(254, 254, 176);
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>318</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
QPushButton:hover:!pressed
{
border-radius:30px;
 background-color:rgba(50, 50, 50, 75); 
}
QPushButton {
  background-color: rgb(255, 255, 255);
	color:#fefe2a;
}
QPushButton:pressed {
border-radius:30px;
    background-color: rgba(50, 50, 50,100); ;
}
QPushButton:checked
{
border:none;
	background-color: rgba(0, 0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rec.qrc">:/img/2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_up</zorder>
   <zorder>pushButton_int</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_conv</zorder>
   <zorder>pushButton_eq</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_CE</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_AC</zorder>
   <zorder>pushButton_H</zorder>
   <zorder>pushButton_M</zorder>
   <zorder>pushButton_S</zorder>
   <zorder>label_down</zorder>
  </widget>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
